### PR TITLE
2D: Add tests and PR comments

### PR DESCRIPTION
**Pull request by StephanyPeralta**
This deliverable contains:

- Test for components and hooks (some doubts here)
- PR suggestions
- Styling on date input icon

### CODE REVIEW

File: __tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable react/display-name */
```
Comment (Raullg98) [inline __tests__/index.test.tsx]: **Quick question**

What is this rule?

File: __tests__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  });
+
+  it("renders Loader icon when isLoading is true", async () => {
+    useNasaSearchMock.mockImplementation(() => ({
+      data,
+      isLoading: true,
+      error: null,
+    }));
+
+    renderHook(() => useNasaSearch(data.date));
```
Comment (Raullg98) [inline __tests__/index.test.tsx]: **Question**

Is this line necessary?

File: __tests__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  });
+
+  it("renders Error Message when an error ocurrs", async () => {
+    useNasaSearchMock.mockImplementation(() => ({
+      data,
+      isLoading: false,
+      error: "Error Test",
+    }));
+
+    renderHook(() => useNasaSearch(data.date));
```
Comment (Raullg98) [inline __tests__/index.test.tsx]: Same question as above

File: package.json [hunk 4]
```diff
@@ -23,7 +23,8 @@
   "devDependencies": {
     "@testing-library/jest-dom": "5.16.1",
     "@testing-library/react": "12.1.2",
-    "@testing-library/user-event": "13.5.0",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/user-event": "^13.5.0",
```
Comment (Raullg98) [inline package.json:27]: Nice!

File: __tests__/index.test.tsx (hunk 1 above)
Comment (StephanyPeralta) [reply __tests__/index.test.tsx]: I have some questions in this test that I want to review with you tomorrow :)

File: __tests__/index.test.tsx (hunk 2 above)
Comment (StephanyPeralta) [reply __tests__/index.test.tsx]: This has been fixed!

File: __tests__/index.test.tsx (hunk 3 above)
Comment (StephanyPeralta) [reply __tests__/index.test.tsx]: This has been fixed!

File: __tests__/index.test.tsx (hunk 1 above)
Comment (StephanyPeralta) [reply __tests__/index.test.tsx]: This has been fixed!

**Pull request merged**